src/BUILD_WINDOWS.md

```diff
@@ -10,14 +10,7 @@ You need to install the following software to build SoftEther VPN for Windows.
 - Microsoft Windows XP, Vista, 7, 8 or later.
 - Microsoft Visual Studio 2008 with the latest SP (SP1 9.0.30729.4462 QFE).
   Make sure that you installed the x64 compiler and build tools.
-
-* Note:
-  Visual Studio 2008 SP1 is required to build SoftEther VPN on Windows.
-  Please make sure that VS2008 'SP1' is installed.
-  Visual Studio 2010, 2012 or 2013 is currently not supported.
-  Visual Studio 2008 Express Edition is not supported.
-  Standard Edition, Professional Edition, Team System or Team Suite is
-  required.

```

Comment:
I haven't tested this, but can you tell me whether really Other versions of Visual studio are currently supported? I can not for example compile SE on VS 2008 Express

Reply:
AppVeyor has Visual C++ 2008 Espress, as you can see on the [Build Environment](https://www.appveyor.com/docs/build-environment/#visual-studio-2008) page.

Reply:
I don't doubt that. My question was whether you can compile SE using MSVC 2008 Express? So far I didn't manage to do it, and I'm downloading the MSVC 2008 + SP1

Reply:
I didn't try to install Visual Studio 2008 Espress locally, but it should work without any problems.

What problem are you encountering?

Reply:
It is complaining about missing afxres.h. Googling a bit shows that this header is part of the MFC which is not included in MSVC 2008 Express. But Stackoverflow also says that replacing this header file with windows.h should solve the problem.

My point is, until we actually are able to compile SE under each MSVC successfully, we should not remove the warning under BUILD_WINDOWS.md

Reply:
MFC is part of the Windows Driver Kit 7.1.0, did you install it in the default path?

Reply:
Let me try that. Give me a couple of minutes

Reply:
I'm not really a windows guy :(

Reply:
No problem.

Reply:
You were right. My bad. After installing windows driver kit, the project is compiling

Reply:
The Windows Driver Kit folders were already added as include and library directories in some projects, such as **Neo6**, prior to #498:
https://github.com/SoftEtherVPN/SoftEtherVPN/blob/94e2e37cc18a6f705eda1a6056b4737feb4f7d02/src/Neo6/Neo6.vcproj#L51

I don't know whether Visual Studio 2008 automatically installs Windows Driver Kit, but according to the previous directory name (`7600.16385.0`) it's an older version.